exercises/anagram/description.md

```diff
@@ -1,8 +1,21 @@
 # Description
 
-An anagram is a rearrangement of letters to form a new word.
-Given a word and a list of candidates, select the sublist of anagrams of the given word.
+An anagram is a rearrangement of letters to form a new word:
```

Comment:
Our [Markdown style guide](https://github.com/exercism/docs/blob/main/building/markdown/markdown.md) has a [one sentence per line rule](https://github.com/exercism/docs/blob/main/building/markdown/markdown.md#one-sentence-per-line).
It is true that not many exercises do yet follow this guideline, but it might be good to apply that rule to this PR. 
See https://github.com/exercism/problem-specifications/pull/1911/files for an example.